fluent-package/templates/package-scripts/fluent-package/deb/postrm

```diff
@@ -30,21 +40,41 @@ if [ "$1" = "purge" ]; then
 		rm -rf $target_dir
 	    fi
 	done
+}
 
+purge_users() {
 	if getent passwd _<%= service_name %>; then
 	    userdel --remove --force _<%= service_name %>
 	fi
 	if getent passwd <%= compat_service_name %>; then
 	    userdel --remove --force <%= compat_service_name %>
 	fi
-fi
-if [ ! "$1" = "upgrade" ]; then
+}
+
+purge_symlinks() {
```

Comment:
This name seems a little confusing since there are other symlinks for the config and the logs.
More specific name such as `purge_bin_symlinks` would be better I think.

Reply:
Thanks, I've fixed it.